Makefile

```diff
@@ -11,7 +11,7 @@ KUBEBUILDER_VERSION=2.3.2
 $(shell mkdir -p "$(OUT)")
 
 test: _test/kubebuilder
-	go test -v .
+	go test -v . -coverprofile coverage.out
```

Comment:
maybe build a html coverage report for local development?  
like this perhaps: `go tool cover -html=coverage.out -o coverage.html`

Reply:
I deliberately left that out, as it clutters the CI and the workspace for everyone, if you don't care about it. I think any developer who needs it can easily add the conversion to their workflow.

Reply:
Hm... I'd like that in Makefile, having it in `.gitignore` prevents it being listed in `git status` and co and temporary files are a thing .. would a subdirectory `out` to put that stuff make you any happier?

Reply:
I'll add it, I don't care that much about having it in or not^^

Reply:
thanks \o/